### PR TITLE
ODC-7359: Fix shipwright build e2e tests

### PR DESCRIPTION
**Pull request by sanketpathak**
Issue: https://issues.redhat.com/browse/ODC-7359
Epic: https://issues.redhat.com/browse/ODC-7358

Test Setup:
```
export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.jephilli-4-11-02-23-0643.devcluster.openshift.com/
export BRIDGE_KUBEADMIN_PASSWORD
export BRIDGE_BASE_ADDRESS
oc login https://api.jephilli-4-10-02-23-0643.devcluster.openshift.com:6443/ -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
  
#Execute below commands under console folder
oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
  
./test-cypress.sh -p shipwright
```

Test Execution Screenshot:
Screenshot 2023-10-31 at 2 59 43 AM
Screenshot 2023-10-29 at 4 58 19 AM

### CODE REVIEW

**Comment by jerolimov** [comment]
Can we also re-enable it in `test-cypress.sh`, or is it still to flaky or slow?

/cc @The-Anton

File: frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts [hunk 1]
```diff
@@ -113,19 +113,21 @@ Then(
...
-  cy.byLegacyTestID(`${buildName}`).should('be.visible').click();
+  cy.get(`[data-test-id='${buildName}'][href*='Build/${buildName}']`).should('be.visible').click();
   cy.get(buildPO.shipwrightBuild.shipwrightBuildRunsTab).should('be.visible').click();
 });
 
 When('user has a failed build run', () => {
   cy.exec(`oc apply -n ${Cypress.env('NAMESPACE')} -f testData/builds/shipwrightBuildRun.yaml`, {
     failOnNonZeroExit: false,
+  }).then(function (result) {
+    cy.log(result.stdout);
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts:124]: No this is just to show the cli result in cypress logs

[the remark this answers is not in the document]
Comment (jerolimov) [reply frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts:124]: But above is a `failOnNonZeroExit: false` :smirk: 

But we can also create a tech debt issue for that and try to remove all `failOnNonZeroExit: false` in the next release.

**Comment by jerolimov** [comment]
In general, this is great. Please check why the tests are broken.

/approve

File: frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts:124]: I've not changed it so as to not stop the execution  of further statements but also see logs in cypress logs

**Comment by jerolimov** [comment]
/retitle ODC-7359: Fix shipwright build e2e tests

@sanketpathak We can not reenable this because it depends on the Pipelines operator and this isn't available currently. When you revert the change that enable this tests we can merge the rest anyway. Wdyt?

**Comment by jerolimov** [comment]
Because it just updates the integration tests

/label acknowledge-critical-fixes-only

**Comment by The-Anton** [comment]
All tests are passing.

/lgtm

**Pull request merged**